scan-command/src/main/java/io/ballerina/scan/internal/CoreRule.java

```diff
@@ -30,7 +31,9 @@
  * @since 0.1.0
  * */
 enum CoreRule {
-    AVOID_CHECKPANIC(RuleFactory.createRule(1, "Avoid checkpanic", RuleKind.CODE_SMELL));
+    AVOID_CHECKPANIC(RuleFactory.createRule(1, RuleDescription.AVOID_CHECKPANIC, RuleKind.CODE_SMELL)),
```

Comment:
On further thought, do we really need the RuleDescription constants? I think we just have the description in line here.